src/components/matches/listing/listings.component.js

```diff
@@ -166,154 +167,174 @@ class Listings extends Component {
       this.setState({ sortAnchorEl: null });
     };
 
-    return (
-      <>
-        {this.props.location.pathname === "/MyAccount" ? (
-          <div className="account-listing-buttons-container">
-            <div className="account-listing-button">
-              <NavLink to="/CreateListing">Create Listing</NavLink>
-            </div>
-            {/* show drafts */}
-            <div className="account-switch-container">
-              <p>Show Drafts</p>
-              <Switch
-                checked={this.state.filters.showDrafts}
-                onChange={() =>
-                  this.setState((prevState) => ({
-                    filters: {
-                      ...prevState.filters,
-                      showDrafts: !this.state.filters.showDrafts,
-                    },
-                  }))
-                }
+    const user = this.props.userInfo;
+    if (user.isVerified === false) {
```

Comment:
Could we change this to `if (!user.isVerified)` instead? There are some instances where `isVerified`is not a field for the user so we should just check for any falsy values

Reply:
fixed!